Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -428,14 +428,52 @@ namespace AZ::Reflection
                 // Cache attributes for the current node. Attribute data will be used in ReflectionAdapter.
                 CacheAttributes();
 
-                // Inherit the change notify attribute from our parent
-                const Name changeNotify = Name("ChangeNotify");
-                if (auto changeNotifyValue = Find(changeNotify); !changeNotifyValue || changeNotifyValue->IsNull())
+                // Inherit the change notify attribute from our parent, if it exists
+                const auto changeNotifyName = DocumentPropertyEditor::Nodes::PropertyEditor::ChangeNotify.GetName();
+                auto parentValue = Find(Name(), changeNotifyName, parentData);
```

Comment:
Will this find the first ancestor with a value, or just the immediate parent?

Reply:
I think it is just the immediate parent. But at the time we're processing the here node, the parent node should have already been processed. So the current node should also inherit all values from its ancestry I guess.

Reply:
That makes sense, thanks.

Reply:
exactly, we all only inherit from our parents, but that doesn't mean that I don't have my great grandfather's right elbow, which everyone knows I totally do.